styles/common.mapcss

```diff
@@ -47,6 +47,12 @@ way|z9-[railway=razed]["razed:usage"="main"][!"razed:service"]["razed:railway"!=
 way|z9-[railway=razed]["usage"="main"][!"service"]["razed:railway"!="light_rail"]["razed:railway"!="subway"]["razed:railway"!="tram"],
 way|z10-[railway=razed]["razed:railway"!="tram"],
 way|z11-[railway=razed]["razed:railway"="tram"]
+way|z9-[railway=dismantled]["dismantled:usage"="branch"][!"dismantled:service"]["dismantled:railway"!="light_rail"]["dismantled:railway"!="subway"]["dismantled:railway"!="tram"],
```

Comment:
I don't think this is worth it, dismantled:railway is barely used and dismantled:usage is never used, so just scrap that.